modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/SelectSubscription.jsx

```diff
@@ -109,6 +109,8 @@ const SelectSubscription = ({
 		return filteredTypes?.productKey;
 	}, [typesProduct, productGroupName, selectedKeyType]);
 
+	const isSingleComplimentaryKey = hasComplimentaryKey && state?.activationKeys.length === 1 ? true : false;
```

Comment:
I believe `state?.activationKeys.length === 1` should give you a boolean already, so you should need the ternary expression after

Reply:
or maybe you need just `==` instead of `===`

Reply:
Hi Ryan, regarding the use of `==` eslint is configured like this: `Expected `===` and instead saw `==``. And this standard is applied to the entire CP project.

Reply:
@samuelleolopes does `state?.activationKeys.length === 1` return a boolean?

Reply:
@kylembischof Yes, I'll send you a correction. I'm investigating an error in the console that I hadn't seen to send it along.